File: php/06for/for21.php
<?php
$fact = 1;
$sum = 0;
$N = mt_rand(1, 50);
echo "Ҳудуди давр: ".$N."<br>";
for($i = 1; $i <= $N; $i++){
	$fact *= $i;
	$sum += 1 / $fact;
}
echo "Сумма = ".number_format($sum, 2);
echo "<br>e(1) = ".number_format(exp(1), 2);
?>

File: cpp/06for/For21.cpp
#include <iostream>
using namespace std;

int main()
{
	//Task("For21");
	int N;
	cin >> N;
	double fact = 1, sum = 0;
	for (int i = 0; i <= N; i++)
	{
		sum += 1.0 / fact;
		fact *= i+1;
	}
	cout << sum;
	
	return 0;
}
